src/solver.cpp

```diff
@@ -296,6 +296,9 @@ inline SolverRegistrar::SolverRegistrar(IdRegistryData& registry)
     RegisterWithSolver(
         registry, ++id, ConvHipImplicitGemmBwdDataV4R1{}, miopenConvolutionAlgoImplicitGEMM);
 
+    RegisterWithSolver(
+        registry, ++id, ConvHipImplicitGemmBwdDataV4R1Xdlops{}, miopenConvolutionAlgoImplicitGEMM);
```

Comment:
[Notice] If you would like to change the order ("priority"), please modify the list in the mlo_dir_conv.cpp.

Reply:
I will collect inception data to determine change order or not

Reply:
the latest commit do not change order

Reply:
@asleepzzz This is to register a solver, by giving it an unique ID, this ID will be used in find-db and perf-db etc. A new solver should always put at the very end of the list, so it doesn't change the IDs of existing solvers.

The order of the solver here will not affect which solver will be picked.